docs/ios/deployment/overview.md

```diff
@@ -0,0 +1,193 @@
+
+## Validate package settings
+
+Every iOS app specifies a unique package identifier and a version. These identifiers are generally set in your project file, from where they are copied to your *Info.plist* file that's located in your project folder at _Platforms\\iOS\\Info.plist_.
+
+Your project file must declare `<ApplicationId>` and `ApplicationVersion` within a `<PropertyGroup>` node. These items should have been generated for you when the project was created. Just validate that they exist and are set to valid values:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <!-- other settings -->
+
+    <PropertyGroup>
+        <ApplicationId>com.companyname.mymauiapp</ApplicationId>
+        <ApplicationVersion>1</ApplicationVersion>
+    </PropertyGroup>
+
+</Project>
+```
+
+The following table describes how each project setting maps to the *Info.plist* file:
+
+| Project setting | Info.plist manifest field |
+| --- | --- |
+| `ApplicationId` | Bundle Identifier |
+| `ApplicationVersion` | Build |
+
+## Connect Visual Studio 2022 to your Mac build host
+
+Building native iOS apps using .NET MAUI requires access to Apple's build tools, which only run on a Mac. Because of this, Visual Studio 2022 must connect to a network-accessible Mac to build .NET MAUI iOS apps. For more information, see [Pair to Mac for iOS development](../pair-to-mac.md).
+
+> [!NOTE]
+> The first time Pair to Mac logs into a Mac build host from Visual Studio 2022, it sets up SSH keys. With these keys, future logins will not require a username or password.
+
+## Publish
+
+At this time, publishing is only supported through the .NET command line interface.
+
+To publish your app, open a terminal and navigate to the project's folder. Run the `dotnet publish` command, providing the following parameters:
+
+| Parameter                    | Value                                                                                           |
+|------------------------------|-------------------------------------------------------------------------------------------------|
+| `-f` or `--framework`        | The target framework, which is `net6.0-maccatalyst`.                                            |
+| `-c` or `--configuration`    | The build configuration, which is `Release`.                                                    |
+
+In addition, the following common parameters can be specified on the command line if they aren't provided in a `<PropertyGroup>` in your project file:
+
+| Parameter                    | Value                                                                                           |
+|------------------------------|-------------------------------------------------------------------------------------------------|
+| `/p:RuntimeIdentifier` | The runtime identifier (RID) for the project. Use `ios-arm64`. |
```

Comment:
You don't actually use this one in the sample below. Not sure if that was intended.